dataactvalidator/config/sqlrules/b29_award_financial_1.sql

```diff
@@ -0,0 +1,9 @@
+-- For File C PriorYearAdjustment must be one of X, B, P, or Blank
+SELECT
+    row_number,
+    prior_year_adjustment,
+    display_tas AS "uniqueid_TAS"
+FROM award_financial
+WHERE (submission_id = {0}
+    AND UPPER(COALESCE(prior_year_adjustment, '')) NOT IN ('X', 'B', 'P', '')
+);
```

Comment:
Can you also remove the extra parens around the entire `WHERE` for both rules? We'll update the styleguide such that parens should only be used as logical groupings within the `WHERE`, not the entire `WHERE` itself.